docs/Getting_Started.md

```diff
@@ -2,106 +2,116 @@
 
 Hokusai aims to provide a smooth CI workflow on a per-project basis.  As such, it is designed to be configured and run from a shell within a project's git repo.
 
-We will assume you have already installed Hokusai and run `hokusai configure`, as detailed in [Setup](#Setup).
+We will assume you have already installed Hokusai and run `hokusai configure`, as detailed in [Setup](../README.md#Setup).
 
-1) Set up Hokusai for a Rails project
-  ```bash
-  cd ./path/to/my/rails/project/git/repo
-  hokusai setup --aws-account-id 12345 --project-type ruby-rails
-  ```
+1) Setup Hokusai for a Rails project
 
-  Note: If you set the environment variable `AWS_ACCOUNT_ID` in your shell, you can omit the `--aws-account-id` option.
+```bash
+cd ./path/to/my/rails/project/git/repo
+hokusai setup --aws-account-id 12345 --project-type ruby-rails
+```
 
-  `hokusai setup` will create:
-  - A `Dockerfile` in your project root
-  - A configuration folder `./hokusai`.  This folder contains:
-      * `./hokusai/config.yml` - contains Hokusai project configuration
-      * `./hokusai/common.yml` - a Docker Compose YAML file imported by `development.yml` and `test.yml`
-      * `./hokusai/development.yml`- a Docker Compose YAML file for the development environment
-      * `./hokusai/test.yml` - a Docker Compose YAML file for the test environment
-      * `./hokusai/staging.yml` - a Kubernetes YAML file for the staging environment
-      * `./hokusai/production.yml` - a Kubernetes YAML file for the production environment
-  - An ECR repository for your project
+Note: If you set the environment variable `AWS_ACCOUNT_ID` in your shell, you can omit the `--aws-account-id` option.
 
-  The files in `./hokusai` as well as the `Dockerfile` are meant to be a starting point for development of your specific application's dependencies, and can / should be freely modified, as you customize your Docker build, add service dependencies to your environments, introduce environment variables, or change the container runtime commands.
+`hokusai setup` will create:
+- A `Dockerfile` in your project root
+- A configuration folder `./hokusai`.  This folder contains:
+  * `./hokusai/config.yml` - Hokusai project configuration
+  * `./hokusai/common.yml` - a Docker Compose YAML file imported by `development.yml` and `test.yml`
+  * `./hokusai/development.yml`- a Docker Compose YAML file for the development environment
+  * `./hokusai/test.yml` - a Docker Compose YAML file for the test environment
+  * `./hokusai/staging.yml` - a Kubernetes YAML file for the staging environment
+  * `./hokusai/production.yml` - a Kubernetes YAML file for the production environment
```

Comment:
You don't need to use the same relative path each time, just file names (`config.yml`) will do.